Kitodo/src/main/java/org/kitodo/production/forms/createprocess/ProcessDataTab.java

```diff
@@ -0,0 +1,236 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.production.forms.createprocess;
+
+import de.unigoettingen.sub.search.opac.ConfigOpac;
+import de.unigoettingen.sub.search.opac.ConfigOpacDoctype;
+
+import java.io.FileNotFoundException;
+import java.util.List;
+import java.util.Objects;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.kitodo.exceptions.ProcessGenerationException;
+import org.kitodo.production.helper.Helper;
+import org.kitodo.production.process.TiffHeaderGenerator;
+import org.kitodo.production.process.TitleGenerator;
+import org.omnifaces.util.Ajax;
+
+public class ProcessDataTab {
+
+    private static final Logger logger = LogManager.getLogger(ProcessDataTab.class);
+
+    private CreateProcessForm createProcessForm;
+    private String docType;
+    private String atstsl = "";
+    private String titleDefinition;
+    private String tifHeaderImageDescription = "";
+    private String tifHeaderDocumentName = "";
```

Comment:
I think it should be `tiff...` at least thats how it is used in the rest of the code